Tests/AuthTests/RequestsTests.swift

```diff
@@ -395,7 +403,7 @@ final class RequestsTests: XCTestCase {
 
     let configuration = AuthClient.Configuration(
       url: clientURL,
-      headers: ["apikey": "dummy.api.key", "X-Client-Info": "gotrue-swift/x.y.z"],
+      headers: ["Apikey": "dummy.api.key", "X-Client-Info": "gotrue-swift/x.y.z"],
```

Comment:
I ended up changing all of the references to `apikey` in the tests to `Apikey` since there is some annoying casing differences generated in the snapshots across platform. This felt acceptable as HTTP headers are specified to be case-insensitive and this was just a quirk of the tests.